lib/typing/parse.go

```diff
@@ -30,6 +25,11 @@ func ParseValue(settings Settings, key string, optionalSchema map[string]KindDet
 		}
 	}
 
+	if val == nil {
```

Comment:
Since `parseValue` is going to return `Invalid` for a `nil` value do we need this check?

Reply:
Good point, let's remove.